src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -83,6 +85,12 @@ const promoTexts: { [promoType: string]: string } = {
     for that facility. To get an accurate Rt, enter case counts for previous
     days by clicking the number of cases on the right (in red) to update case
     numbers.`,
+  newModelInputs: `New! We’ve updated our model to factor in recoveries and 
+    deaths to enable more accurate projections based on active cases. This 
+    may have caused some of your projection values to change. You can input 
+    recoveries and deaths when you “Add Historical Data” for a facility.
```

Comment:
Nit but the double quotes in here are not the "standard" double quotes and may render strangely in some browsers, if I recall. Usually you get these when they are copied and pasted from somewhere, but we should replace them.

Reply:
good catch, thanks!